### ISSUE TITLE
PATCH request acts as POST request

### ISSUE THREAD

**Issue by natn2323** [djmnz/RestFluencing#17]
I found that sending PATCH requests using the RestFluencing library actually sends POST requests. Doing some investigating, I found this:

```csharp
/// <summary>
/// Prepare a PATCH request to the URL. You must have set the property <see cref="RestConfiguration.BaseUrl"/> of the configuration
/// </summary>
public static RestRequest **Patch**(this RestConfiguration config, string relativeUrl)
{
	if (config == null)
	{
		throw new ArgumentNullException(nameof(config), ErrorMessages.NoConfiguration);
	}

	if (config.BaseUrl == null)
	{
		throw new ArgumentException(ErrorMessages.BaseUrlIsNotSet, nameof(relativeUrl));
	}

	return Rest.**Post**(relativeUrl, config);
}
```

Was this a typo, or was this intentional and PATCH requests aren't currently supported?

**Comment by natn2323** [comment]
Yeah, I've corrected the line and will make a pull request.

**Comment by djmnz** [comment]
Thanks I will merge it in 😁

**Comment by djmnz** [comment]
Released as version 1.1.1.48
